service/model/user_role.go

```diff
@@ -7,22 +7,25 @@ import (
 type UserRole string
 
 const (
-	UserRoleAdmin UserRole = "admin"
-	UserRoleUser  UserRole = "user"
+	UserRoleAdmin   UserRole = "admin"
+	UserRoleManager UserRole = "manager"
+	UserRoleUser    UserRole = "user"
```

Comment:
New role `Manager` introduces. Why?

- Admin  role - have access to organization settings (implemented in #14) and can create projects as well

- Manager role - does not have access to organization settings, but can create projects _(role for Engineering managers in STRV)_

Other naming option would be `SuperAdmin` and `Admin roles`.

If you approve the addition of this new role, I will also create an endpoint that enables admin users to assign the manager role to other users.

Reply:
Interesting point! I'd definitely steer towards something more simple and obvious, for example:
- org admin
- project admin
- user
How about other similar platforms? What names are they using?

And do we expect to have more different items?

If yes, consider simply having users and rights (read/modify org/project).